modules/apps/account/account-admin-web/package.json

```diff
@@ -1,14 +1,16 @@
 {
 	"dependencies": {
 		"frontend-js-web": "*",
-		"metal-dom": "2.16.8",
-		"metal-events": "2.16.8"
+		"metal-path-parser": "1.0.4",
+		"metal-router": "3.6.3",
+		"metal-state": "2.16.8",
+		"senna": "2.7.9"
 	},
-	"name": "account-admin-web",
+	"name": "portal-portlet-bridge-soy-impl",
```

Comment:
Why this package name change?

Reply:
I didn't change it myself, it was one of conflicted changes that I didn't catch in git, well I did catch it but I probably got confused by "ours", "theirs", and whatever rubbish language used to differentiate between current and conflicted change.

Bottom line, it shouldn't have been changed.